.github/workflows/main.yml

```diff
@@ -32,7 +32,11 @@ jobs:
         uses: actions/setup-python@v2
         with:
             python-version: ${{ matrix.python-version }}
-
+      
+      # before install
+      - name: before install
+        run: |
+          pip install libpulse0:i386
```

Comment:
The right way of doing this might be to add the library into the dependencies in `setup.cfg`. Try doing that maybe?

Reply:
Will try!